Sources/AudioKitUI/Visualizations/FFTView.swift

```diff
@@ -36,12 +36,8 @@ class FFTModel: ObservableObject {
         // loop by two through all the fft data
         for i in stride(from: 0, to: FFT_SIZE - 1, by: 2) {
             if i / 2 < numberOfBars {
```

Comment:
i think these lines are still an issue. using a stride of 2 means that we're losing half the data, and then we lose even more by only selecting the data at indexes smaller than the number of bars.

to be honest, i'm not sure what the best way of fixing this is. i tried aggregating some of the frequency bins into an array of size `numberOfBars`. that will show you everything up to 22kHz (which i think is the max given the 44kHz sample rate), but may be less useful visually since the higher frequencies don't get hit much.

this looks something like:

```swift
        let bins_per_bar = Int((Float(fftData.count) / Float(numberOfBars)).rounded(.up))
        
        for i in 0..<fftData.count {
            let amplitude = // calculate amplitude here
            let barIndex = i / bins_per_bar
            
            tempAmplitudes[barIndex] += amplitude * (1 / Float(bins_per_bar))
        }
```

Reply:
Maybe we could try reducing the `FFT_SIZE` in `FFTView`. That's the only thing I can think of.

Reply:
@aburgel See if #24 helps at all or seems like a good idea. It might not be the best solution, but it's something to think about.